src/components/mobile-payment-link/MobilePaymentLink.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div class="mobile-payment-link">
+    <div class="header">
```

Comment:
Isn't `div class="header"` a bit of an anti-pattern? Shouldn't the `header` html tag be used to be semantically correct?